src/ui-lib/sass/04-objects/_locations-layout.scss

```diff
@@ -0,0 +1,19 @@
+.grav-o-locations-layout {
+  @media (min-width: gravy-breakpoint(regular)) {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
missing closed braket.
you want to have:
```
@media {
}
@media {
} 
```
since they specify `min-width`, the larger will get settings from the smaller.

Reply:
whoops